modules/client-configuration/pages/registration-cli.adoc

```diff
@@ -18,8 +18,9 @@ Use the bootstrap script procedure to register traditional clients.
 For more information, see xref:registration-bootstrap.adoc[].
 ====
 
-This procedure requires that you have installed the [package]``salt-minion`` package on the Salt client before registration.
+This procedure requires that you have installed the [package]``venv-salt-minion`` package (Salt bundle) on the Salt client before registration.
```

Comment:
I think we need to specify both `venv-salt-minion` or `salt-minion` as we are not shipping venv-salt-minion for all of the systems yet. And it's better to list the client tools containing the Salt Bundle. Am I right? @juliogonzalez

Reply:
thanks, submitted locally.